### ISSUE TITLE
MIT License

### ISSUE THREAD

**Issue by Hussainiglobal** [uxsolutions/bootstrap-datepicker#2407]
Hello,

I just want to confirm that is this comes with MIT license? and we can use it in MVC application

Thanks,

**Comment by McNetic** [comment]
Obviously, you did not look at the license link on the project front page. It clearly says Apache 2.0 License: https://github.com/uxsolutions/bootstrap-datepicker/blob/master/LICENSE.
